### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by claucece**

### CODE REVIEW

File: net-im/coyim/Makefile [hunk 1]
```diff
@@ -6,7 +6,7 @@ DISTVERSION=	0.3.11
 CATEGORIES=	net-im
 
 MAINTAINER=	santhosh.raju@gmail.com
-COMMENT=	Safe and secure chat client
+COMMENT=	A safe and secure by default chat client
```
Comment (fraggerfox) [inline net-im/coyim/Makefile]: Due to the rules about `COMMENT` in FreeBSD port, a comment should not start with "A" / "An" or "The".

https://www.freebsd.org/doc/en/books/porters-handbook/book.html#makefile-comment

**Review by fraggerfox — CHANGES_REQUESTED** [review]
If you can remove the "A" from the comment, it should comply with the FreeBSD ports `COMMENT` description guidelines. Then I can go ahead and merge these changes into the repository.

The rest looks good.

Thanks!!

File: net-im/coyim/Makefile (hunk 1 above)
Comment (claucece) [reply net-im/coyim/Makefile]: Solved ;)

Comment (fraggerfox) [reply net-im/coyim/Makefile]: LGTM, pulling in the changes.

Thank you.

**Pull request merged**